### ISSUE TITLE
hd5 support

### ISSUE THREAD

**Issue by cardillo** [cardillo/joinery#1]

**Comment by harsham4026** [comment]
@cardillo I was wondering if you are supporting the HDFS (Hadoop Distributed File System) for reading and writing the data. I was planning to add the support for reading and writing the data to Hadoop Distributed File System. Shall I add the support for Hadoop Distributed File System? If you say yes I'll add the code for supporting Hadoop Distributed File System and then for HDF5.

**Comment by harsham4026** [comment]
@cardillo yes HDFS is just like another file system for reading and writing the files. I have written some API's for reading and writing the data to HDFS. I was wondering how we can plugin the HDFS API's and support for reading and writing from HDFS as well. I'll raise a PR for the API's I have written. Please let me know how we can integrate that to existing code part. Please give some code example.
